vanilla-app/package.json

```diff
@@ -17,20 +17,22 @@
     "vite": "4.5.2"
   },
   "dependencies": {
-    "@justeattakeaway/pie-button": "0.45.2",
+    "@justeattakeaway/pie-assistive-text": "^0.2.0",
+    "@justeattakeaway/pie-button": "0.45.3",
     "@justeattakeaway/pie-card": "0.17.2",
-    "@justeattakeaway/pie-cookie-banner": "0.17.0",
+    "@justeattakeaway/pie-chip": "^0.1.0",
```

Comment:
do we want to pin this?

Reply:
And pie-assistive-text, in case you miss it 👀